src/main/java/io/tiledb/cloud/rest_api/Login.java

```diff
@@ -0,0 +1,73 @@
+package io.tiledb.cloud.rest_api;
```

Comment:
Should we move this up a level so we keep autogenerated in a separate package?

Reply:
Done!